version: "3.8"

services:
  wb_data_service:
    container_name: "wb_data_service"
    build:
      dockerfile: ./docker/Dockerfile.wb_data_service
    ports:
      - "8000:8000"
    networks:
      - wb
  wb_data_tg_bot:
    container_name: "wb_data_tg_bot"
    build:
      dockerfile: ./docker/Dockerfile.wb_data_tg_bot
    ports:
      - "8005:8005"
    env_file:
      - .env
    networks:
      - wb
  wb_db:
    container_name: "wb_db"
    image: postgres:15.5-alpine
    restart: on-failure
    environment:
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=postgres
      - POSTGRES_DB=wb_db
      - POSTGRES_HOST=localhost
      - POSTGRES_PORT=5432
    volumes:
      - ./wb_db_data:/var/lib/postgresql/data
    ports:
      - "5435:5432"
    networks:
      - wb

networks:
  wb:
    driver: bridge
